index.html

```diff
@@ -77,33 +78,33 @@ <h5 class='offcanvas-title' id='offcanvasExampleLabel'>Menu</h5>
         <div class='offcanvas-body'>
           <ul class="offcanvas-menu  " >
             <li class="" >
-              <a class="btn btn-secundary" href="">
-                  <button class="btn btn-primary">About</button>     
+              <a class="" type="button" href="./views/about.html">
+                  <button class="btn btn-primary" type="button">About</button>     
               </a>                         
             </li>            
             <li class="" >
-              <a class="btn btn-secundary" href="">
-                  <button class="btn btn-primary">Program</button>     
+              <a class="" type="button" href="">
+                  <button class="btn btn-primary" type="button">Program</button>     
               </a>                         
             </li>            
             <li class="" >
-              <a class="btn btn-secundary" href="">
-                  <button class="btn btn-primary">Join</button>     
+              <a class="" type="button" href="">
+                  <button class="btn btn-primary" type="button">Join</button>     
               </a>                         
             </li>            
             <li class="" >
-              <a class="btn btn-secundary" href="">
-                  <button class="btn btn-primary">Sponsor</button>     
+              <a class="" type="button" href="">
+                  <button class="btn btn-primary" type="button">Sponsor</button>     
               </a>                         
             </li>            
             <li class="" >
-              <a class="btn btn-secundary" href="">
-                  <button class="btn btn-primary">News</button>     
+              <a class="" type="button" href="">
+                  <button class="btn btn-primary" type="button">News</button>     
```

Comment:
- [ ] Please in the about page the footer should has a dark background as in the design.
![image](https://user-images.githubusercontent.com/74744787/226133737-16772dbd-25c8-49fb-a7a0-364a9ada97ad.png)
Your design 
![image](https://user-images.githubusercontent.com/74744787/226133770-b28912c9-bb2c-4c98-bdc5-86526203ec9f.png)

Reply:
done